Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1321,6 +1320,9 @@ WarpXOpenPMDPlot::WriteOpenPMDFieldsAll ( //const std::string& filename,
         series_iteration.setTime( time );
     }
 
+    // If there are no fields to be written, interrupt the function here
+    if ( varnames.size()==0 ) return;
```

Comment:
This is needed in order to be able to gracefully handle the case where there is no field.

Reply:
Can you try to move this further up in the function? Or does the particle write not work if we don't do the open ops in the lines above?